packages/nimble-components/src/label-provider/core/index.ts

```diff
@@ -29,7 +31,9 @@ const supportedLabels = {
     popupIconInformation: popupIconInformationLabel,
     filterSearch: filterSearchLabel,
     filterNoResults: filterNoResultsLabel,
-    loading: loadingLabel
+    loading: loadingLabel,
+    scrollLeftLabel, // 'scrollLeft' is already reserved in the base class
+    scrollRightLabel
```

Comment:
Can we use an alternate name to avoid the discrepancy (i.e. the other attributes avoid label in the names)?

Left and Right might not be great long-term because of [ltr/rtl issues](https://github.com/openui/open-ui/issues/522) either. Some ideas, scrollForward/scrollBackward, scrollNext/scrollPrevious (the [tabs aria pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) seems to refer to the concepts as next and previous but not sure how much to read into that in terms of i18n)

Reply:
Just saw a push for the name of `scrollLeftButtonLabel` but I think I'd still suggest one of the other alternatives because:
1. Same rtl/ltr concerns
2. the labels shouldn't be coupled to control usage, i.e. button, in their naming

Reply:
Went with 'scrollForward'/'scrollBackward'.